app/models/mentor/student.rb

```diff
@@ -6,12 +6,15 @@ class Student
       '_application_coding_level',
       '_application_code_samples',
       '_application_learning_history',
-      '_application_skills',
-      '_application_code_background'
+      '_application_language_learning_period',
+      '_application_skills'
+      
     ]
 
     attr_accessor :coding_level, :code_samples
-    attr_accessor :learning_history, :code_background
```

Comment:
Seeing this, I would probably have kept both. Is `code_background` *too much* or *too little* information for the mentors? Or is it just to personal?